pyproject.toml

```diff
@@ -4,7 +4,6 @@ version = "0.1.0"
 description = "This repo serves the model so that it can be queried."
 authors = ["Your Name <you@example.com>"]
 readme = "README.md"
-packages = [{include = "model_service"}]
```

Comment:
I notice that here you remove this line but you add it again in an upcoming commit. Why is that?

Reply:
Because I thought that the GitHub Action build failed due to the addition of the packages link in `pyproject.toml` whereas the opposite turned out to be true.

Reply:
Because I thought that the GitHub Action build failed due to the addition of the packages link in `pyproject.toml` whereas the opposite turned out to be true.

Reply:
Got it, ok. I'll drop these commits then, since that didn't seem to be a problem